democracy/models/base.py

```diff
@@ -116,6 +118,13 @@ class Meta:
         abstract = True
 
 
+class Organization(BaseModel):
+    name = models.CharField(verbose_name=_('name'), max_length=255)
+    admin_users = models.ManyToManyField(
+        User, blank=True, related_name='admin_organizations'
```

Comment:
Just use `settings.AUTH_USER_MODEL` directly here -- aliasing it to `User` makes the casual reader think it's not pluggable.